pymoose/pymoose/predictors/predictor.py

```diff
@@ -0,0 +1,75 @@
+import abc
```

Comment:
Thanks for the review. What's very strange it's that I did rename the file with `git mv`. See the the first commit here: https://github.com/tf-encrypted/moose/pull/1115/commits/25a5f95dbeebca4bffa1a544d2828f0554f60ee2

But after I added the `AesWrapper` to the renamed file `predictor.py` the diff got broken (I tried twice because I really wanted to keep the link). Do you have any idea what could have happened?